--- origsrc/Python-3.11.0a3/setup.py	2022-01-02 23:06:10.183826500 +0100
+++ src/Python-3.11.0a3/setup.py	2022-01-02 23:48:03.086866500 +0100
@@ -1661,14 +1661,8 @@ class PyBuildExt(build_ext):
                 # OpenBSD and FreeBSD use Tcl/Tk library names like libtcl83.a,
                 # but the include subdirs are named like .../include/tcl8.3.
                 dotversion = dotversion[:-1] + '.' + dotversion[-1]
-            tcl_include_sub = []
-            tk_include_sub = []
-            for dir in self.inc_dirs:
-                tcl_include_sub += [dir + os.sep + "tcl" + dotversion]
-                tk_include_sub += [dir + os.sep + "tk" + dotversion]
-            tk_include_sub += tcl_include_sub
-            tcl_includes = find_file('tcl.h', self.inc_dirs, tcl_include_sub)
-            tk_includes = find_file('tk.h', self.inc_dirs, tk_include_sub)
+            tcl_includes = find_file('tcl.h', self.inc_dirs, [])
+            tk_includes = find_file('tk.h', self.inc_dirs, [])
 
         if (tcllib is None or tklib is None or
             tcl_includes is None or tk_includes is None):
