--- origsrc/Python-3.13.0/configure.ac	2024-10-07 07:02:14.000000000 +0200
+++ src/Python-3.13.0/configure.ac	2024-11-17 04:12:43.850158400 +0100
@@ -260,12 +260,6 @@ AC_DEFINE([_NETBSD_SOURCE], [1],
           [Define on NetBSD to activate all library features])
 
 # The later definition of _XOPEN_SOURCE and _POSIX_C_SOURCE disables
-# certain features on FreeBSD, so we need __BSD_VISIBLE to re-enable
-# them.
-AC_DEFINE([__BSD_VISIBLE], [1],
-          [Define on FreeBSD to activate all library features])
-
-# The later definition of _XOPEN_SOURCE and _POSIX_C_SOURCE disables
 # certain features on Mac OS X, so we need _DARWIN_C_SOURCE to re-enable
 # them.
 AC_DEFINE([_DARWIN_C_SOURCE], [1],
