--- origsrc/Python-3.8.7/Python/dynload_shlib.c	2020-12-21 17:25:24.000000000 +0100
+++ src/Python-3.8.7/Python/dynload_shlib.c	2021-01-12 14:49:06.902341400 +0100
@@ -35,6 +35,8 @@
 
 const char *_PyImport_DynLoadFiletab[] = {
 #ifdef __CYGWIN__
+    "." SOABI ".dll",
+    ".abi" PYTHON_ABI_STRING ".dll",
     ".dll",
 #else  /* !__CYGWIN__ */
     "." SOABI ".so",
--- origsrc/Python-3.11.0b1/Misc/platform_triplet.c	2022-05-23 18:43:15.849895100 +0200
+++ src/Python-3.11.0b1/Misc/platform_triplet.c	2022-05-23 18:47:45.802287100 +0200
@@ -968,6 +968,14 @@
 #  endif
 #elif defined(__VXWORKS__)
 PLATFORM_TRIPLET=vxworks
+#elif defined(__CYGWIN__)
+# if defined(__x86_64__)
+PLATFORM_TRIPLET=x86_64-cygwin
+# elif defined(__i386__)
+PLATFORM_TRIPLET=i386-cygwin
+# else
+#   error unknown platform triplet
+# endif
 #elif defined(__wasm32__)
 #  if defined(__EMSCRIPTEN__)
 PLATFORM_TRIPLET=wasm32-emscripten