--- origsrc/Python-3.8.10/Modules/ossaudiodev.c	2021-05-03 11:47:56.000000000 +0200
+++ src/Python-3.8.10/Modules/ossaudiodev.c	2021-05-20 11:37:45.886398900 +0200
@@ -959,7 +959,7 @@ static PyGetSetDef oss_getsetlist[] = {
 };
 
 static PyTypeObject OSSAudioType = {
-    PyVarObject_HEAD_INIT(&PyType_Type, 0)
+    PyVarObject_HEAD_INIT(NULL, 0)
     "ossaudiodev.oss_audio_device", /*tp_name*/
     sizeof(oss_audio_t),        /*tp_basicsize*/
     0,                          /*tp_itemsize*/
@@ -993,7 +993,7 @@ static PyTypeObject OSSAudioType = {
 };
 
 static PyTypeObject OSSMixerType = {
-    PyVarObject_HEAD_INIT(&PyType_Type, 0)
+    PyVarObject_HEAD_INIT(NULL, 0)
     "ossaudiodev.oss_mixer_device", /*tp_name*/
     sizeof(oss_mixer_t),            /*tp_basicsize*/
     0,                              /*tp_itemsize*/
